resources/multisite_drupal_standard.make

```diff
@@ -153,6 +153,12 @@ projects[css_injector][patch][] = https://www.drupal.org/files/issues/css_inject
 
 projects[ctools][subdir] = "contrib"
 projects[ctools][version] = "1.9"
+; Deprecate PHP 4 style constructors for PHP 7 compatibility
+; https://www.drupal.org/node/2528736
+; The patch corresponds to commit 96e56f523ce3d9609e677fcc6263cfc3e798b0ab in
+; the ctools repository.
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-11467
+projects[ctools][patch][] = patches/ctools-php7-2528736.patch
```

Comment:
Why do you use a file related with a commit instead of the "real" patch (https://www.drupal.org/files/issues/deprecating_php4_style-2528736-23.patch)?
The commit you use seems to suffer from this bug: https://www.drupal.org/node/2657834